src/packages/src/xroad/common/base/usr/share/xroad/scripts/_setup_memory.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+die () {
+  echo >&2 "$@"
+  exit 1
+}
+
+function to_megabytes() {
+  if [[ $1 =~ ^([0-9]+)m$ ]]; then
+    echo "${BASH_REMATCH[1]}"
+  elif [[ $1 =~ ^([0-9]+)g$ ]]; then
+    echo "${BASH_REMATCH[1]} * 1024" | bc
+  else
+    echo ""
+  fi
+}
+
+get_params_line() {
+  grep --color=never "^${1}=" "/etc/xroad/services/local.properties"
+}
+
+apply_memory_config(){
+  local -r params_file="/etc/xroad/services/local.properties"
+  local -r params="$(get_params_line "$1")"
+
+  local -r xms="-Xms$2"
+  local -r xmx="-Xmx$3"
+
+  if [ -z "$params" ]; then
+    echo "$1=$xms $xmx" >> "$params_file"
+  else
+    local pattern="-Xms[0-9]+[mg]"
```

Comment:
According to documentation, Xmx and Xms can also be set in k or in plain bytes.  Also unit suffix should be case insensitive.